### PR TITLE
Feature/UptimeRobot

### PR DESCRIPTION
**Pull request by MedAliMarz**
This PR include a node for UptimeRobot with support for:

- Account: getAccountDetails:
- Monitor: create reset getAll update delete
- Alert Contact: create getAll update delete
- Maintenance Window: create getAll update delete
- Public Status Page: create getAll update delete

<details>
<summary>
the next workflow can be used for reviewing the node functionalities
</summary>

```json
{
  "nodes": [
    {
      "parameters": {},
      "name": "Start",
      "type": "n8n-nodes-base.start",
      "typeVersion": 1,
      "position": [
        250,
        300
      ]
    },
    {
      "parameters": {},
      "name": "UptimeRobot",
      "type": "n8n-nodes-base.uptimeRobot",
      "typeVersion": 1,
      "position": [
        410,
        20
      ],
      "credentials": {
        "uptimeRobotApi": "Dali UptimeRobotApi creds"
      }
    },
    {
      "parameters": {
        "resource": "monitor",
        "operation": "create",
        "friendly_name": "=Monitor{{Date.now()}}",
        "type": 1,
        "url": "http://n8n.io"
      },
      "name": "UptimeRobot1",
      "type": "n8n-nodes-base.uptimeRobot",
      "typeVersion": 1,
      "position": [
        410,
        290
      ],
      "credentials": {
        "uptimeRobotApi": "Dali UptimeRobotApi creds"
      }
    },
    {
      "parameters": {
        "resource": "monitor",
        "operation": "reset",
        "id": "={{$node[\"UptimeRobot1\"].json[\"id\"]}}"
      },
      "name": "UptimeRobot2",
      "type": "n8n-nodes-base.uptimeRobot",
      "typeVersion": 1,
      "position": [
        580,
        290
      ],
      "credentials": {
        "uptimeRobotApi": "Dali UptimeRobotApi creds"
      }
    },
    {
      "parameters": {
        "resource": "monitor",
        "limit": 1,
        "filters": {}
      },
      "name": "UptimeRobot3",
      "type": "n8n-nodes-base.uptimeRobot",
      "typeVersion": 1,
      "position": [
        740,
        290
      ],
      "credentials": {
        "uptimeRobotApi": "Dali UptimeRobotApi creds"
      }
    },
    {
      "parameters": {
        "resource": "monitor",
        "operation": "update",
        "id": "={{$node[\"UptimeRobot1\"].json[\"id\"]}}",
        "updateFields": {
          "friendly_name": "UpdatedMonitor",
          "url": "http://community.n8n.io"
        }
      },
      "name": "UptimeRobot4",
      "type": "n8n-nodes-base.uptimeRobot",
      "typeVersion": 1,
      "position": [
        890,
        290
      ],
      "credentials": {
        "uptimeRobotApi": "Dali UptimeRobotApi creds"
      }
    },
    {
      "parameters": {
        "resource": "monitor",
        "limit": 1,
        "filters": {
          "monitors": "={{$node[\"UptimeRobot1\"].json[\"id\"]}}"
        }
      },
      "name": "UptimeRobot5",
      "type": "n8n-nodes-base.uptimeRobot",
      "typeVersion": 1,
      "position": [
        1050,
        290
      ],
      "credentials": {
        "uptimeRobotApi": "Dali UptimeRobotApi creds"
      }
    },
    {
      "parameters": {
        "resource": "monitor",
        "operation": "delete",
        "id": "={{$node[\"UptimeRobot1\"].json[\"id\"]}}"
      },
      "name": "UptimeRobot6",
      "type": "n8n-nodes-base.uptimeRobot",
      "typeVersion": 1,
      "position": [
        1220,
        290
      ],
      "credentials": {
        "uptimeRobotApi": "Dali UptimeRobotApi creds"
      }
    },
    {
      "parameters": {
        "resource": "alertContact",
        "operation": "create",
        "friendly_name": "=AlertContact{{Date.now()}}",
        "type": 2,
        "value": "=fakeemail{{Date.now().toString().substr(6)}}@test.com"
      },
      "name": "UptimeRobot7",
      "type": "n8n-nodes-base.uptimeRobot",
      "typeVersion": 1,
      "position": [
        410,
        450
      ],
      "credentials": {
        "uptimeRobotApi": "Dali UptimeRobotApi creds"
      }
    },
    {
      "parameters": {
        "resource": "alertContact",
        "limit": 1,
        "filters": {}
      },
      "name": "UptimeRobot8",
      "type": "n8n-nodes-base.uptimeRobot",
      "typeVersion": 1,
      "position": [
        560,
        450
      ],
      "credentials": {
        "uptimeRobotApi": "Dali UptimeRobotApi creds"
      }
    },
    {
      "parameters": {
        "resource": "alertContact",
        "operation": "update",
        "id": "={{$node[\"UptimeRobot7\"].json[\"id\"]}}",
        "updateFields": {
          "friendly_name": "UpdatedAlertContact"
        }
      },
      "name": "UptimeRobot9",
      "type": "n8n-nodes-base.uptimeRobot",
      "typeVersion": 1,
      "position": [
        700,
        450
      ],
      "credentials": {
        "uptimeRobotApi": "Dali UptimeRobotApi creds"
      }
    },
    {
      "parameters": {
        "resource": "alertContact",
        "operation": "delete",
        "id": "={{$node[\"UptimeRobot7\"].json[\"id\"]}}"
      },
      "name": "UptimeRobot10",
      "type": "n8n-nodes-base.uptimeRobot",
      "typeVersion": 1,
      "position": [
        850,
        450
      ],
      "credentials": {
        "uptimeRobotApi": "Dali UptimeRobotApi creds"
      }
    },
    {
      "parameters": {
        "resource": "mwindows",
        "operation": "create",
        "duration": 5,
        "friendly_name": "=Mwindow{{Date.now()}}",
        "start_time": "2021-05-20T22:00:00.000Z",
        "type": 2,
        "value": "20:30"
      },
      "name": "UptimeRobot11",
      "type": "n8n-nodes-base.uptimeRobot",
      "typeVersion": 1,
      "position": [
        410,
        600
      ],
      "credentials": {
        "uptimeRobotApi": "Dali UptimeRobotApi creds"
      }
    },
    {
      "parameters": {
        "resource": "mwindows",
        "limit": 1,
        "filters": {}
      },
      "name": "UptimeRobot12",
      "type": "n8n-nodes-base.uptimeRobot",
      "typeVersion": 1,
      "position": [
        550,
        600
      ],
      "credentials": {
        "uptimeRobotApi": "Dali UptimeRobotApi creds"
      }
    },
    {
      "parameters": {
        "resource": "mwindows",
        "operation": "update",
        "id": "={{$node[\"UptimeRobot11\"].json[\"id\"]}}",
        "duration": 4,
        "updateFields": {
          "type": 3,
          "value": "1-2"
        }
      },
      "name": "UptimeRobot13",
      "type": "n8n-nodes-base.uptimeRobot",
      "typeVersion": 1,
      "position": [
        700,
        600
      ],
      "credentials": {
        "uptimeRobotApi": "Dali UptimeRobotApi creds"
      }
    },
    {
      "parameters": {
        "resource": "mwindows",
        "limit": 1,
        "filters": {
          "mwindows": "={{$node[\"UptimeRobot11\"].json[\"id\"]}}"
        }
      },
      "name": "UptimeRobot14",
      "type": "n8n-nodes-base.uptimeRobot",
      "typeVersion": 1,
      "position": [
        850,
        600
      ],
      "credentials": {
        "uptimeRobotApi": "Dali UptimeRobotApi creds"
      }
    },
    {
      "parameters": {
        "resource": "mwindows",
        "operation": "delete",
        "id": "={{$node[\"UptimeRobot11\"].json[\"id\"]}}"
      },
      "name": "UptimeRobot15",
      "type": "n8n-nodes-base.uptimeRobot",
      "typeVersion": 1,
      "position": [
        1000,
        600
      ],
      "credentials": {
        "uptimeRobotApi": "Dali UptimeRobotApi creds"
      }
    },
    {
      "parameters": {
        "resource": "psp",
        "operation": "create",
        "friendly_name": "=Psp{{Date.now()}}",
        "monitors": "={{$node[\"UptimeRobot1\"].json[\"id\"]}}",
        "additionalFields": {}
      },
      "name": "UptimeRobot16",
      "type": "n8n-nodes-base.uptimeRobot",
      "typeVersion": 1,
      "position": [
        550,
        130
      ],
      "credentials": {
        "uptimeRobotApi": "Dali UptimeRobotApi creds"
      }
    },
    {
      "parameters": {
        "resource": "psp",
        "limit": 1,
        "filters": {
          "psps": "={{$node[\"UptimeRobot16\"].json[\"id\"]}}"
        }
      },
      "name": "UptimeRobot17",
      "type": "n8n-nodes-base.uptimeRobot",
      "typeVersion": 1,
      "position": [
        700,
        130
      ],
      "credentials": {
        "uptimeRobotApi": "Dali UptimeRobotApi creds"
      }
    },
    {
      "parameters": {
        "resource": "psp",
        "operation": "update",
        "id": "={{$node[\"UptimeRobot16\"].json[\"id\"]}}",
        "updateFields": {
          "friendly_name": "UpdatedPSP"
        }
      },
      "name": "UptimeRobot18",
      "type": "n8n-nodes-base.uptimeRobot",
      "typeVersion": 1,
      "position": [
        850,
        130
      ],
      "credentials": {
        "uptimeRobotApi": "Dali UptimeRobotApi creds"
      }
    },
    {
      "parameters": {
        "resource": "psp",
        "operation": "delete",
        "id": "={{$node[\"UptimeRobot16\"].json[\"id\"]}}"
      },
      "name": "UptimeRobot19",
      "type": "n8n-nodes-base.uptimeRobot",
      "typeVersion": 1,
      "position": [
        1000,
        130
      ],
      "credentials": {
        "uptimeRobotApi": "Dali UptimeRobotApi creds"
      }
    }
  ],
  "connections": {
    "Start": {
      "main": [
        [
          {
            "node": "UptimeRobot1",
            "type": "main",
            "index": 0
          },
          {
            "node": "UptimeRobot7",
            "type": "main",
            "index": 0
          },
          {
            "node": "UptimeRobot11",
            "type": "main",
            "index": 0
          },
          {
            "node": "UptimeRobot",
            "type": "main",
            "index": 0
          }
        ]
      ]
    },
    "UptimeRobot1": {
      "main": [
        [
          {
            "node": "UptimeRobot16",
            "type": "main",
            "index": 0
          }
        ]
      ]
    },
    "UptimeRobot2": {
      "main": [
        [
          {
            "node": "UptimeRobot3",
            "type": "main",
            "index": 0
          }
        ]
      ]
    },
    "UptimeRobot3": {
      "main": [
        [
          {
            "node": "UptimeRobot4",
            "type": "main",
            "index": 0
          }
        ]
      ]
    },
    "UptimeRobot4": {
      "main": [
        [
          {
            "node": "UptimeRobot5",
            "type": "main",
            "index": 0
          }
        ]
      ]
    },
    "UptimeRobot5": {
      "main": [
        [
          {
            "node": "UptimeRobot6",
            "type": "main",
            "index": 0
          }
        ]
      ]
    },
    "UptimeRobot7": {
      "main": [
        [
          {
            "node": "UptimeRobot8",
            "type": "main",
            "index": 0
          }
        ]
      ]
    },
    "UptimeRobot8": {
      "main": [
        [
          {
            "node": "UptimeRobot9",
            "type": "main",
            "index": 0
          }
        ]
      ]
    },
    "UptimeRobot9": {
      "main": [
        [
          {
            "node": "UptimeRobot10",
            "type": "main",
            "index": 0
          }
        ]
      ]
    },
    "UptimeRobot11": {
      "main": [
        [
          {
            "node": "UptimeRobot12",
            "type": "main",
            "index": 0
          }
        ]
      ]
    },
    "UptimeRobot12": {
      "main": [
        [
          {
            "node": "UptimeRobot13",
            "type": "main",
            "index": 0
          }
        ]
      ]
    },
    "UptimeRobot13": {
      "main": [
        [
          {
            "node": "UptimeRobot14",
            "type": "main",
            "index": 0
          }
        ]
      ]
    },
    "UptimeRobot14": {
      "main": [
        [
          {
            "node": "UptimeRobot15",
            "type": "main",
            "index": 0
          }
        ]
      ]
    },
    "UptimeRobot16": {
      "main": [
        [
          {
            "node": "UptimeRobot17",
            "type": "main",
            "index": 0
          }
        ]
      ]
    },
    "UptimeRobot17": {
      "main": [
        [
          {
            "node": "UptimeRobot18",
            "type": "main",
            "index": 0
          }
        ]
      ]
    },
    "UptimeRobot18": {
      "main": [
        [
          {
            "node": "UptimeRobot19",
            "type": "main",
            "index": 0
          }
        ]
      ]
    },
    "UptimeRobot19": {
      "main": [
        [
          {
            "node": "UptimeRobot2",
            "type": "main",
            "index": 0
          }
        ]
      ]
    }
  }
}
```

</details>

### CODE REVIEW

File: packages/nodes-base/credentials/UptimeRobotApi.credentials.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import {
+	ICredentialType,
+	NodePropertyTypes,
+} from 'n8n-workflow';
+
+
+export class UptimeRobotApi implements ICredentialType {
+	name = 'uptimeRobotApi';
+	displayName = 'UptimeRobotApi';
```
Comment (ivov) [inline packages/nodes-base/credentials/UptimeRobotApi.credentials.ts:9]:
```suggestion
	displayName = 'Uptime Robot API';
```

File: packages/nodes-base/credentials/UptimeRobotApi.credentials.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+	documentationUrl = 'uptimerobot';
```
Comment (ivov) [inline packages/nodes-base/credentials/UptimeRobotApi.credentials.ts:10]:
```suggestion
	documentationUrl = 'uptimeRobot';
```

File: packages/nodes-base/nodes/UptimeRobot/GenericFunctions.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import { OptionsWithUri } from 'request';
+import {
+	IExecuteFunctions,
+	IExecuteSingleFunctions,
+	ILoadOptionsFunctions,
+} from 'n8n-core';
+import { IDataObject, NodeApiError, NodeOperationError, } from 'n8n-workflow';
```
Comment (ivov) [inline packages/nodes-base/nodes/UptimeRobot/GenericFunctions.ts:7]: Multinline imports.

File: packages/nodes-base/nodes/UptimeRobot/GenericFunctions.ts [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+export async function uptimeRobotApiRequest(this: IExecuteFunctions | IExecuteSingleFunctions | ILoadOptionsFunctions, method: string, resource: string, body: any = {}, qs: IDataObject = {}, uri?: string, option: IDataObject = {}): Promise<any> { // tslint:disable-line:no-any
```
Comment (ivov) [inline packages/nodes-base/nodes/UptimeRobot/GenericFunctions.ts:9]: Suggestion: Every async function returns a `Promise` and an unspecified return type implies `any`, so you can remove `Promise<any>` while preserving type information, including removal of the TSLint exception.

**Review by ivov — CHANGES_REQUESTED** [review]
Thanks for taking this on 👍 

Quick code review - node not yet tested.

File: packages/nodes-base/nodes/UptimeRobot/UptimeRobot.node.ts [hunk 5]
```diff
@@ -0,0 +1,544 @@
...
+	publicStatusPagesOperations,
+} from './PublicStatusPagesDescription';
+
+import * as moment from 'moment';
+
+export class UptimeRobot implements INodeType {
+	description: INodeTypeDescription = {
+		displayName: 'UptimeRobot',
+		name: 'uptimeRobot',
+		icon: 'file:uptimerobot.png',
```
[the remark this answers is not in the document]
Comment (MedAliMarz) [reply packages/nodes-base/nodes/UptimeRobot/UptimeRobot.node.ts:38]: I think that's an old logo, maybe we can keep using the current one

File: packages/nodes-base/nodes/UptimeRobot/GenericFunctions.ts [hunk 6]
```diff
@@ -15,20 +17,20 @@ export async function uptimeRobotApiRequest(this: IExecuteFunctions | IExecuteSi
...
 	};
 	options = Object.assign({}, options, option);
-	if (Object.keys(options.body).length === 0) {
-		delete options.body;
-	}
 	try {
-		return await this.helpers.request!(options);
+		const responseData = await this.helpers.request(options);
+		if (responseData.stat !== 'ok') {
+			throw new NodeApiError(this.getNode(),responseData.error);
```
Comment (ivov) [inline packages/nodes-base/nodes/UptimeRobot/GenericFunctions.ts:31]: I would think `NodeOperationError` is preferable for these cases, but there are examples of both in the codebase - see [Slack](https://github.com/n8n-io/n8n/blob/5c65a73cc21ac4348eaae6b6b010980ff789cefe/packages/nodes-base/nodes/Slack/GenericFunctions.ts#L60-L62) vs. [ActiveCampaign](https://github.com/n8n-io/n8n/blob/4801510b0cbc86390f8e3ca8eef008c2880f14ec/packages/nodes-base/nodes/ActiveCampaign/GenericFunctions.ts#L55-L57).

Thoughts welcome @BHesseldieck 

Also, no need to specify the `error` property in `responseData.error` - it should be found.

File: packages/nodes-base/nodes/UptimeRobot/MonitorDescription.ts [hunk 7]
```diff
@@ -443,11 +438,11 @@ export const monitorFields = [
 				default: '',
 				options: [
 					{
-						name: 'paused',
+						name: 'Paused',
```
Comment (ivov) [inline packages/nodes-base/nodes/UptimeRobot/MonitorDescription.ts:441]:
```suggestion
						name: 'Pause',
```

File: packages/nodes-base/nodes/UptimeRobot/UptimeRobot.node.ts [hunk 8]
```diff
@@ -36,16 +40,16 @@ export class UptimeRobot implements INodeType {
 		displayName: 'UptimeRobot',
 		name: 'uptimeRobot',
 		icon: 'file:uptimerobot.png',
-		group: ['output'],
+		group: [ 'output' ],
```
Comment (ivov) [inline packages/nodes-base/nodes/UptimeRobot/UptimeRobot.node.ts:43]: Codebase convention is no whitespace inside single-line arrays. Same for elsewhere.
```suggestion
		group: ['output'],
```

File: packages/nodes-base/nodes/UptimeRobot/UptimeRobot.node.ts [hunk 9]
```diff
@@ -119,19 +123,19 @@ export class UptimeRobot implements INodeType {
 			/* -------------------------------------------------------------------------- */
 			/*                                Maintenance Windows                         */
```
Comment (ivov) [inline packages/nodes-base/nodes/UptimeRobot/UptimeRobot.node.ts:124]:
```suggestion
			/*                                Maintenance Window                         */
```

File: packages/nodes-base/nodes/UptimeRobot/UptimeRobot.node.ts [hunk 10]
```diff
@@ -119,19 +123,19 @@ export class UptimeRobot implements INodeType {
...
 			/* -------------------------------------------------------------------------- */
-			...maintenanceWindowsOperations,
-			...maintenanceWindowsFields,
+			...maintenanceWindowOperations,
+			...maintenanceWindowFields,
 			/* -------------------------------------------------------------------------- */
 			/*                               Public Status Pages                          */
```
Comment (ivov) [inline packages/nodes-base/nodes/UptimeRobot/UptimeRobot.node.ts:129]:
```suggestion
			/*                               Public Status Page                          */
```

File: packages/nodes-base/nodes/UptimeRobot/UptimeRobot.node.ts [hunk 11]
```diff
@@ -119,19 +123,19 @@ export class UptimeRobot implements INodeType {
...
+			...publicStatusPageOperations,
+			...publicStatusPageFields,
 		],
 	};
 
 	async execute(this: IExecuteFunctions): Promise<INodeExecutionData[][]> {
 		const items = this.getInputData();
-		const returnData: IDataObject[] = [];
+		const returnData = [];
 		const length = items.length as unknown as number;
```
Comment (ivov) [inline packages/nodes-base/nodes/UptimeRobot/UptimeRobot.node.ts:139]:
```suggestion
		const length = items.length;
```

File: packages/nodes-base/nodes/UptimeRobot/UptimeRobot.node.ts [hunk 12]
```diff
@@ -222,141 +203,79 @@ export class UptimeRobot implements INodeType {
...
-				if (resource === 'mwindows') {
+				} else if (resource === 'mwindows') {
 					if (operation === 'create') {
-						const friendlyName = this.getNodeParameter('friendly_name', i) as string;
-						const value = this.getNodeParameter('value', i) as string;
-						const type = this.getNodeParameter('type', i) as number;
-						const duration = this.getNodeParameter('duration', i) as number;
 						const startTime = this.getNodeParameter('start_time', i) as string;
+						const type = this.getNodeParameter('type', i) as number;
 						let parsedStartTime;
```
Comment (ivov) [inline packages/nodes-base/nodes/UptimeRobot/UptimeRobot.node.ts:279]: Another option:
```
const parsedStartTime = type === 1
	? moment(startTime).unix()
	: moment(startTime).format('HH:mm');
```

File: packages/nodes-base/credentials/UptimeRobotApi.credentials.ts [hunk 13]
```diff
@@ -6,8 +6,8 @@ import {
 
 export class UptimeRobotApi implements ICredentialType {
 	name = 'uptimeRobotApi';
-	displayName = 'UptimeRobotApi';
-	documentationUrl = 'uptimerobot';
+	displayName = 'Uptime Robot Api';
```
Comment (ivov) [inline packages/nodes-base/credentials/UptimeRobotApi.credentials.ts:9]:
```suggestion
	displayName = 'Uptime Robot API';
```

File: packages/nodes-base/nodes/UptimeRobot/GenericFunctions.ts [hunk 14]
```diff
@@ -1,12 +1,14 @@
...
 } from 'n8n-core';
-import { IDataObject, NodeApiError, NodeOperationError, } from 'n8n-workflow';
+import {
+	IDataObject,
+	NodeApiError,
+	NodeOperationError,
+} from 'n8n-workflow';
 
-export async function uptimeRobotApiRequest(this: IExecuteFunctions | IExecuteSingleFunctions | ILoadOptionsFunctions, method: string, resource: string, body: any = {}, qs: IDataObject = {}, uri?: string, option: IDataObject = {}): Promise<any> { // tslint:disable-line:no-any
+export async function uptimeRobotApiRequest(this: IExecuteFunctions, method: string, resource: string, body: any = {}, qs: IDataObject = {}, uri?: string, option: IDataObject = {}) {
```
Comment (ivov) [inline packages/nodes-base/nodes/UptimeRobot/GenericFunctions.ts:11]:
```suggestion
export async function uptimeRobotApiRequest(this: IExecuteFunctions, method: string, resource: string, body: IDataObject = {}, qs: IDataObject = {}, uri?: string, option: IDataObject = {}) {
```

**Review by ivov — CHANGES_REQUESTED** [review]

**Comment by ivov** [comment]
Node functionality tested - all operations working.

Minor details to add to the above:
`HTTP(s)` → `HTTP(S)`
`Web-Hook` → `Webhook`
`mwindows` → `mwindow`

File: packages/nodes-base/nodes/UptimeRobot/MonitorDescription.ts [hunk 15]
```diff
@@ -0,0 +1,497 @@
...
+						value: 2,
+					},
+					{
+						name: 'seems down',
+						value: 8,
+					},
+					{
+						name: 'down',
+						value: 9,
+					},
```
Comment (ivov) [inline packages/nodes-base/nodes/UptimeRobot/MonitorDescription.ts:272]: Capital letter for `name` value.

File: packages/nodes-base/nodes/UptimeRobot/PublicStatusPageDescription.ts [hunk 16]
```diff
@@ -0,0 +1,327 @@
...
+						value: 2,
+					},
+					{
+						name: 'status (up-down-paused)',
+						value: 3,
+					},
+					{
+						name: 'status (down-up-paused)',
+						value: 4,
+					},
```
Comment (ivov) [inline packages/nodes-base/nodes/UptimeRobot/PublicStatusPageDescription.ts:130]: Capital letter for `name` value.

File: packages/nodes-base/nodes/UptimeRobot/PublicStatusPageDescription.ts [hunk 17]
```diff
@@ -0,0 +1,327 @@
...
+				],
+				description: 'The sorting of the status page',
```
Comment (ivov) [inline packages/nodes-base/nodes/UptimeRobot/PublicStatusPageDescription.ts:132]:
```suggestion
				description: 'The sorting of the status page.',
```

File: packages/nodes-base/nodes/UptimeRobot/PublicStatusPageDescription.ts [hunk 18]
```diff
@@ -0,0 +1,327 @@
...
+						value: 2,
+					},
+					{
+						name: 'status (up-down-paused)',
+						value: 3,
+					},
+					{
+						name: 'status (down-up-paused)',
+						value: 4,
+					},
```
Comment (ivov) [inline packages/nodes-base/nodes/UptimeRobot/PublicStatusPageDescription.ts:321]: Capital letter for `name` value.

**Review by ivov — APPROVED** [review]

**Comment by RicardoE105** [comment]
Closing this in favor of #1830

**Comment by janober** [comment]
Thanks a lot. Got merged with #1830

**Comment by janober** [comment]
Got released with n8n@0.122.0

**Pull request closed without merging**